PSICover/PSICover.cs

```diff
@@ -163,26 +164,53 @@ class Analyzer {
          var code = File.ReadAllLines (file);
          for (int i = 0; i < code.Length; i++)
             code[i] = code[i].Replace ('<', '\u00ab').Replace ('>', '\u00bb');
+         int blks = blocks.Count;
+         int hitsCount = 0;
          foreach (var block in blocks) {
             bool hit = hits[block.Id] > 0;
-            string tag = $"<span class=\"{(hit ? "hit" : "unhit")}\">";
+            string tag = $"<span class=\"{(hit ? "hit" : "unhit")}\" title=\"{(hit ? hits[block.Id] + " hits" : "")}\">";

```

Comment:
Multiple check for hit. Also, for unhit title will be added which is not necessary.